Gemfile

```diff
@@ -55,6 +55,7 @@ group :development, :test do
   gem 'rspec_junit_formatter'
   gem 'simplecov'
   gem 'shoulda-matchers', require: false
+  gem 'dotenv-rails'
```

Comment:
we use figaro for this. is it not working for you?

Reply:
err...yes. I forgot we were using that.